### ISSUE TITLE
Should this repository be archived?

### ISSUE THREAD

**Issue by ndiego** [WordPress/theme-experiments#313]
This repository has not seen any major updates in years and block theming has come a long way since WordPress 6.0. The examples in this repository are no longer best practice and could lead new theme developers down the "wrong path".

There is a ton of great historical context here, but should this repository be archived and the readme.md updated to indicate that no new development will happen and point folks to other resources like the Create Block Theme plugin instead?

**Comment by justintadlock** [comment]
Agreed. Plus, we have the Community Themes repo now: https://github.com/WordPress/community-themes

**Comment by carolinan** [comment]
Yes lets archive this.
